### PR TITLE
add authentication using external server

### PR DESCRIPTION
**Pull request by maximskripnik**

### CODE REVIEW

**Comment by maximskripnik** [comment]
I forgot to make the authentication with google optional. Let me fix that

File: app/silhouette/SilhouetteModule.scala [hunk 1]
```diff
@@ -79,31 +79,34 @@ class SilhouetteModule extends AbstractModule {
...
+      )
+    }
 
   @Provides
-  def provideSocialProviderRegistry(googleProvider: CustomGoogleProvider): SocialProviderRegistry = {
-    SocialProviderRegistry(Seq(googleProvider))
-  }
+  def provideSocialProviderRegistry(googleProvider: Option[CustomGoogleProvider]): SocialProviderRegistry =
+    SocialProviderRegistry(
+      Seq(
```
Comment (warlockrichard) [inline app/silhouette/SilhouetteModule.scala:106]: How about `googleProvider.to(Seq)`?

**Review by warlockrichard — CHANGES_REQUESTED** [review]
Everything else is OK to me.

File: app/silhouette/SilhouetteModule.scala (hunk 1 above)
Comment (maximskripnik) [reply app/silhouette/SilhouetteModule.scala:106]: This is a template to add more providers latter on, starting from twitter, facebook and vk in the https://github.com/o360/backend/issues/9

Comment (warlockrichard) [reply app/silhouette/SilhouetteModule.scala:106]: Ok, then `.flatten`?

Comment (maximskripnik) [reply app/silhouette/SilhouetteModule.scala:106]: I don't see a difference at all, but if you find this more readable, let's do it this way

Comment (warlockrichard) [reply app/silhouette/SilhouetteModule.scala:106]: Thanks

**Review by warlockrichard — APPROVED** [review]

**Pull request merged**